LvivCompany.Bookstore/LvivCompany.Bookstore.Web/Views/Home/Index.cshtml

```diff
@@ -1,102 +1,35 @@
-﻿<div class="container">
-    <div class="row">
-        
-        <div class="col-md-3 col-sm-6">
-            <span class="thumbnail">
-                <img src="http://placehold.it/500x400" alt="...">
-                <h4>Product Tittle</h4>
-                <div class="ratings">
-                    <span class="glyphicon glyphicon-star"></span>
-                    <span class="glyphicon glyphicon-star"></span>
-                    <span class="glyphicon glyphicon-star"></span>
-                    <span class="glyphicon glyphicon-star"></span>
-                    <span class="glyphicon glyphicon-star-empty"></span>
-                </div>
-                <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. </p>
-                <hr class="line">
-                <div class="row">
-                    <div class="col-md-6 col-sm-6">
-                        <p class="price">$29,90</p>
-                    </div>
-                    <div class="col-md-6 col-sm-6">
-                        <button class="btn btn-success right"> BUY ITEM</button>
-                    </div>
+@model List<BookDetailViewModel>
```

Comment:
It's better to add a collection of books to `HomeViewModel` and reference `HomeViewModel` here instead of `List<BookDetailViewModel>`